src/Serilog.AspNetCore/AspNetCore/RequestLoggingMiddleware.cs

```diff
@@ -70,9 +70,10 @@ public async Task Invoke(HttpContext httpContext)
 
         bool LogCompletion(HttpContext httpContext, DiagnosticContextCollector collector, int statusCode, double elapsedMs, Exception ex)
         {
+            var logger = Log.Logger.ForContext<RequestLoggingMiddleware>();
```

Comment:
This will cause some additional allocations on every request; alternatively, we could create a:

```csharp
static readonly LogEventProperty SourceContextProperty = new LogEventProperty("SourceContext", typeof(SerilogRequestLoggingMiddleware).FullName);
```

and then include this in the list of properties that are manually added to the event a few lines below?

Reply:
But do we not need to have set the source context for the IsEnabled chack?
```csharp
if (!logger.IsEnabled(level)) return false;
```
 if not then I agree with you and I'll change it to your suggestion.

Reply:
Ah right you are, nicely spotted 😅

Would still be nice to trim some allocations down the track, but I think the change as written is good for now 👍 

Very minor nitpick:

```
var logger = Log.Logger.ForContext<RequestLoggingMiddleware>();
```

It's usually idiomatically written as:

```
var logger = Log.ForContext<RequestLoggingMiddleware>();
```

Reply:
Alright, I've fixed the nitpick-issue 👍